src/componentsUniversal/header/header.js

```diff
@@ -0,0 +1,36 @@
+import './header.scss'
+import logo from '../../assets/images/logo.png'
+import Button from '../button/button';
+
+const Header = () => {
+    return (
+        <header className="header">
+        <div className="container">
+          <div className="header__top">
+            <img className="header__img" src={logo} alt="logo-img" />
+            <nav className="menu">
+              <ul className="menu__list">
+                <li className="menu__item menu__item-header">
+                  <a href="/" className="menu__link menu__link-header menu__link-first">Courses</a>
```

Comment:
same here: look at react-router and replace `a` tags with it's Link or NavLink.
hint: NavLink is better for this case